s3_project/classes/create_database.py

```diff
@@ -7,10 +7,10 @@
 
 class ProjectDatabase:
     def __init__(self):
-        self.server = '35.179.41.219'
-        self.database = 'SpartaGlobal'
-        self.username = 'SA'
-        self.password = 'Passw0rd2018'
+        self.server = input('Server:\n')
+        self.database = input('Database:\n')
+        self.username = input('UsernameL\n')
+        self.password = input('Password:\n')
```

Comment:
Requiring user input halfway through a potentially automated process is also not great.  You are relying on the user type in an IP address accurately on the first try!